bootstrap.d/media-libs.yml

```diff
@@ -328,7 +331,7 @@ packages:
         - '--buildtype=debugoptimized'
         #- '--buildtype=debug'
         - '-Dglx=dri'
-        - '-Dplatforms=surfaceless,drm,wayland,x11'
+        - '-Dplatforms=wayland,x11'
```

Comment:
Why do we disable the surfaceless backend?

Reply:
It’s not a valid option anymore, but it is still enabled according to the output at the end of configure